### DISCUSSION TITLE
Может сделать Инициализацию плагина автоматической?

### DISCUSSION THREAD

**Discussion by sulfur17** [vanessa-opensource/add#844] [Ideas]
@artbear 
Почти всегда после получения плагина функцией Плагин() в xddTestRunner происходит вызов Плагин.Инициализация()
А имеет ли смысл плагин без инициализации?
Может сделать чтобы Инициализация вызывалась всегда самостоятельно, прям в функции Плагин()?

Иначе в инициализации теста после
`УтвержденияПроверкаТаблиц = КонтекстЯдра.Плагин("УтвержденияПроверкаТаблиц");`
приходится еще писать 
`УтвержденияПроверкаТаблиц.Инициализация(КонтекстЯдраПараметр);`
и так для каждого плагина которому важен контекст ядра для работы.
Это и больше текста и интуитивно кажется что плагин бы должен быть уже инициализирован, т.е. ведет к ошибкам.

Либо, если от этой идеи откажемся, то я хотя бы в документацию добавлю что инициализировать плагины надо явно.

**Comment by artbear** [comment]
Давай с доки начнем, как уже списались в чате )
